### PR TITLE
#594 - add terraform scripts for AWS private VPC with coreOS

### PR DESCRIPTION
**Pull request by tayzlor**

### CODE REVIEW

**Comment by tayzlor** [comment]
Created follow-up https://github.com/Capgemini/Apollo/issues/604

File: bootstrap/aws/public-cloud/config-default.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Keeping atlas variable without prefix as it's been shared by consul and tf at the moment.
+export ATLAS_TOKEN=${ATLAS_TOKEN:?"Need to set ATLAS_TOKEN non-empty"}
+export ATLAS_INFRASTRUCTURE=${ATLAS_INFRASTRUCTURE:-capgemini/apollo}
+
+export TF_VAR_access_key=${TF_VAR_access_key:?"Need to set TF_VAR_access_key non-empty"}
+export TF_VAR_secret_key=${TF_VAR_secret_key:?"Need to set TF_VAR_secret_key non-empty"}
+
+# Overrides default folder in Terraform.py inventory.
+export TF_VAR_STATE_ROOT="${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}"
```
Comment (wallies) [inline bootstrap/aws/public-cloud/config-default.sh]: is this gonna work, since now its terraform/aws/private and public cloud

Comment (tayzlor) [inline bootstrap/aws/public-cloud/config-default.sh]: you will need to set APOLLO_PROVIDER as aws/private-cloud or aws/public-cloud

Comment (wallies) [inline bootstrap/aws/public-cloud/config-default.sh]: ok cool. just need to document that.

**Pull request merged**